test/built-ins/Simd/Simd.operators.js

```diff
@@ -0,0 +1,66 @@
+//  Copyright (C) 2015
+//
+//  This software is provided 'as-is', without any express or implied
+//  warranty.  In no event will the authors be held liable for any damages
+//  arising from the use of this software.
+//
+//  Permission is granted to anyone to use this software for any purpose,
+//  including commercial applications, and to alter it and redistribute it
+//  freely, subject to the following restrictions:
+//
+//  1. The origin of this software must not be misrepresented; you must not
+//     claim that you wrote the original software. If you use this software
+//     in a product, an acknowledgment in the product documentation would be
+//     appreciated but is not required.
+//  2. Altered source versions must be plainly marked as such, and must not be
+//     misrepresented as being the original software.
+//  3. This notice may not be removed or altered from any source distribution.
+
+/*---
+description: Test SIMD operators.
+includes: [simdUtilities.js]
+---*/
+
+function testOperators(type) {
+  var inst = createTestValue(type);
+  throws(function() { Number(inst) });
+  throws(function() { +inst });
+  throws(function() { -inst });
+  throws(function() { ~inst });
+  throws(function() { Math.fround(inst) });
+  throws(function() { inst|0} );
```

Comment:
Spacing is consistent with no spaces for unary operators and spaces for binary operators.